package.json

```diff
@@ -24,6 +34,8 @@
     "lint:js": "eslint . --cache",
     "test": "vitest --coverage",
     "test:eslint7": "pnpm remove eslint; pnpm add eslint@7; pnpm test",
+    "tsc": "tsc",
```

Comment:
You can leave one command, but it seems logical to me to add a separate command for type generation.